src/pudl/metadata/resources/eia.py

```diff
@@ -19,7 +19,21 @@
     "boilers_entity_eia": {
         "description": "Static boiler attributes compiled from the EIA-860 and EIA-923 data.",
         "schema": {
-            "fields": ["plant_id_eia", "boiler_id", "prime_mover_code"],
+            "fields": [
+                "plant_id_eia",
+                "boiler_id",
+                "boiler_operating_date",
+                "boiler_manufacturer",
+                "boiler_manufacturer_code",
+                "firing_rate_using_coal_tons_per_hour",
+                "firing_rate_using_oil_bbls_per_hour",
+                "firing_rate_using_gas_mcf_per_hour",
+                "firing_rate_using_other_fuels",
+                "waste_heat_input_mmbtu_per_hour",
+                "wet_dry_bottom",
+                "fly_ash_reinjection",
+                "prime_mover_code",
```

Comment:
Re: naming. The shared name is not affected by the harvest because the boiler table it originates from doesn't include a generator ID, so we could _technically_ get away with not changing the names but I agree that from a clarity perspective renaming both would be useful.

 In `boiler_fuel_eia923` table from which the boiler `prime_mover_code` originates, one boiler can have more than one `prime_mover_code` associated with it in any given year (one record per plant, boiler and energy source code). Given this information, presumably it wouldn't make sense to harvest this record into either a static or annual table which expects 1:1 boiler attributes, since we'd lose information on 1:m records?

Reply:
Yeah, that definitely sounds correct. What kind of situations do you find multiple prime movers attached to one boiler? Is it in combined cycle natural gas plants?  What's the right way to deal with this? Should we have more than one prime mover in association with each boiler? Several columns like we have for `energy_source_code`?

Does EIA's documentation provide any insight into the meaning of the single prime mover that's associated with the boiler in the `boiler_info` table?  Or is the `prime_mover_code` only coming from the `boiler_fuel_eia923` data table?  If it only appears in the data tables, and it doesn't represent an actual attribute of the boiler, then maybe it shouldn't be getting harvested at all. Again @cmgosnell may have more informed thoughts on this.

Reply:
It's only coming from `boiler_fuel_eia923`. I agree - in the latest iteration (not pushed yet but will be shortly) I've pulled this out of all the entity tables.

Reply:
In the end we're not harvesting prime_mover, so this question is no longer relevant.

Reply:
i'm late to this party but it definitely seems to make sense to not harvest `prime_mover_code` for boilers!